aiida/common/setup.py

```diff
@@ -638,6 +638,10 @@ def create_config_noninteractive(profile='default', force_overwrite=False, dry_r
     # finalizing
     write = not dry_run
     new_profile = update_profile(profile, new_profile, write=write)
+    old_profiles = [p for p in get_profiles_list() if p != profile]
+    if profile=='default' or not old_profiles:
```

Comment:
I think it's safer to just look at ``if not old_profiles`` - even if named default, I think it's better that the user explicitly decides to change the default one here.